bin/node/runtime/src/lib.rs

```diff
@@ -120,11 +120,18 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	// implementation changes and behavior does not, then leave spec_version as
 	// is and increment impl_version.
 	spec_version: 284,
-	impl_version: 0,
+	impl_version: 1,
```

Comment:
I see the pallets major versions increase. It should be spec_version increase in the runtime

Reply:
Incremented spec_version. Reset impl_version